tck/src/test/java/org/eclipse/microprofile/jwt/tck/container/jaxrs/ClaimValueInjectionEndpoint.java

```diff
@@ -146,30 +143,7 @@ else if(jtiValue.equals(jwtID)) {
             .build();
         return result;
     }
-    @GET
-    @Path("/verifyInjectedAudience")
-    @Produces(MediaType.APPLICATION_JSON)
-    public JsonObject verifyInjectedAudience(@QueryParam("aud") String audience) {
```

Comment:
Maybe its me, or my interpretation of what `aud` was for, but it seems like all we're checking is that you support aud.  So what is the issue with just supporting aud (but not verifying it)?

Reply:
Section 4.1 definitely says (said) the JWT itself must have the `aud` claim to be considered an interoperable MP JWT.

In terms of why use the `getClaim(String)` method, that's the pattern we've taken for all other non-required claims so I followed it for consistency.